app/models/friendship.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Friendship < ApplicationRecord
+  belongs_to :user
+  belongs_to :friend, class_name: 'User'
+
+  validates :user, presence: true
```

Comment:
- Note that `belongs_to :user` already adds prsence validation to `user`

Reply:
Ok